specs/linux/ipmi_lan.table

```diff
@@ -0,0 +1,19 @@
+table_name("ipmi_lan")
+description("IPMI LAN information.")
+schema([
+    Column("mc_name", TEXT, "Name of the MC"),
+    Column("mc_id", TEXT, "Device ID of the MC"),
```

Comment:
Hrm, yeah I hadn't thought about that. I'd say either way should be fine, I think perhaps keeping more descriptive names for the `id` might be worth it, as you mention there could be confusion otherwise. It seems like `ipmi_lan` and `ipmi_mc` could share some column names though? To your note above, if the `name` column in the `impi_mc` table matched that of `name` in this table it might be more intuitive for `join`ing purposes?